config/application.yml.default

```diff
@@ -425,6 +427,7 @@ production:
   attribute_encryption_key:
   attribute_encryption_key_queue: '[]'
   aws_logo_bucket: ''
+  component_previews_embed_frame_ancestors: ['https://developers.login.gov']
```

Comment:
Continuing from https://github.com/18F/identity-idp/pull/9786#discussion_r1434195072, I don't think we'll want to override this here, and do so in the live deployed configuration instead.

```suggestion
```